### PR TITLE
more userguide cleanup

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

**Comment by loosebazooka** [comment]
@chanseokoh I think there's too much that can go wrong with the thin jar stuff being described in the faq. Changed it to just describing how to use extraFilesDirectories.

File: USER_GUIDE.md [hunk 1]
```diff
@@ -368,51 +368,19 @@ The plugin defaults to `appengine-web.xml` based deployment if your project cont
...
-### How do I deploy a thin jar to App Engine?
+### How do I deploy other files with my application to App Engine?
 
-You can do this if you have an `app.yaml` based project and you have a custom `entrypoint` configured to start it.
-
-You will need to configure the stage goal to include your project's dependencies. You could use something like the
-[`maven-dependency-plugin:copy-dependency`](https://maven.apache.org/plugins/maven-dependency-plugin/copy-dependencies-mojo.html)
-goal to copy dependencies into a directory and then configure that directory to be staged with your application using
-the `extraFilesDirectories` parameter in the appengine plugin. For example:
+When configuring an `app.yaml` based project, the `extraFilesDirectories` configuration options lets
```
Comment (chanseokoh) [inline USER_GUIDE.md]: ... configuration option lets ...

File: USER_GUIDE.md [hunk 2]
```diff
@@ -368,51 +368,19 @@ The plugin defaults to `appengine-web.xml` based deployment if your project cont
...
-Then when you run `mvn package appengine:deploy` the dependencies are copied and included as part of the deployment.
-
-Since the default entrypoint assumes a fat jar, you must define a custom entrypoint to run this application. In this
-example we might have an `app.yaml` that looks like:
-
-```
-runtime: java11
-entrypoint: java -Xmx64m -cp "*" com.example.MyMainClass
-```
+Then when you run `mvn package appengine:deploy` the extras are staged with you application and included as part of the deployment.
```
Comment (chanseokoh) [inline USER_GUIDE.md]: ... the extra files are ... ?

**Comment by chanseokoh** [comment]
Yeah, I agree there are too many things that can go wrong.

**Pull request merged**